tools/syz-showprio/showprio.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+// Tool used to visualize the priorities table.
```

Comment:
Package documentation should be placed on top of the `package main`. This is standard convention used by tooling as well. E.g. see:
https://godoc.org/github.com/google/syzkaller/pkg/csource

The standard convention is to start it with utility name: "syz-showprio visualizes the priorities table".
Also extend it a bit. There can be other priorities. This is call to call priorities form the prog package.